docs/markdown/snippets/add_release_note_snippets_here

```diff
@@ -1,3 +1,5 @@
 ## Added `cpp_std` option for the Visual Studio C++ compiler
 Allows the use of C++17 features and experimental not-yet-standardized
 features. Valid options are `c++11`, `c++14`, `c++17`, and `c++latest`.
+
+## Fixed Renesas RX Family compiler to work with latest meson, added working cross file, fixed assembly file bug
```

Comment:
Please remove this. Release notes should only list new functionality, not bug fixes. The old text there is wrong and should not be there. I removed in trunk so you need to rebase.